src/components/ContentHighlights/ContentHighlightsCardItemsContainer.jsx

```diff
@@ -21,19 +17,43 @@ const ContentHighlightsCardItemsContainer = () => {
         xl: 4,
       }}
     >
-      {highlightCourses.map(({
-        uuid, title, contentType, authoringOrganizations,
+      {highlightedContent.map(({
+        uuid, title, contentType, authoringOrganizations, contentKey,
       }) => (
+
         <ContentHighlightCardItem
+          isLoading={isLoading}
           key={uuid}
           cardImageUrl="https://picsum.photos/200/300"
           title={title}
-          contentType={contentType.toLowerCase()}
+          hyperlink={generateAboutPageUrl(enterpriseSlug, contentType?.toLowerCase(), contentKey)}
+          contentType={contentType?.toLowerCase()}
           partners={authoringOrganizations}
         />
       ))}
+      {isLoading && <div data-testid="card-item-skeleton" />}
```

Comment:
[curious] Should this be rendered higher up in the component? Example:

```jsx
const ContentHighlightsCardItemsContainer = ({ enterpriseSlug, isLoading, highlightedContent }) => {
  if (isLoading) {
    // render loading skeletons
  }

  if (!highlightedContent || highlightedContent?.length === 0) {
    // render reasonable empty state UI, e.g. warning `Alert`
  }

  return (
    <CardGrid>
      {highlightedContent.map(({
        uuid, title, contentType, authoringOrganizations, contentKey,
      }) => { /* ... */ })}
    </CardGrid>
  );
```

Reply:
generally, loading skeletons should mimic the general styles for the content that will once loading is complete, in this case skeleton cards.

Perhaps re-use/adapt the following from `HighlightStepperConfirmContent`:

```jsx
return (
  <div data-testid="skeleton-container">
    <CardGrid
      columnSizes={{
        xs: 12,
        md: 6,
        lg: 4,
        xl: 3,
      }}
    >
      {[...new Array(8)].map((element, index) => <SkeletonContentCard key={index} />)}
    </CardGrid>
  </div>
);
```

Reply:
Decided to extract it out, could be a utility component in the future for cards, but basically created a new component called `SkeletonContentCardContainer` which contains the logic `HighlightStepperConfirmContent`

`
import PropTypes from 'prop-types';
import SkeletonContentCard from './SkeletonContentCard';

const SkeletonContentCardContainer = ({ length }) => [
  ...new Array(length),
// eslint-disable-next-line react/no-array-index-key
].map((_, index) => <SkeletonContentCard key={index} />);

SkeletonContentCardContainer.propTypes = {
  length: PropTypes.number.isRequired,
};

export default SkeletonContentCardContainer;
`
This results in a much cleaner final implementation. Was tempted to add a `{children}` prop to map into instead of `SkeletonContentCard` but in this iteration I thought to keep it simple for now.

Final resulting condition is as such
`
  if (isLoading) {
    return (
      <CardGrid columnSizes={HIGHLIGHTS_CARD_GRID_COLUMN_SIZES}>
        <SkeletonContentCardContainer length={MAX_CONTENT_ITEMS_PER_HIGHLIGHT_SET} />
      </CardGrid>
    );
  }
  `